src/typeDefs/index.ts

```diff
@@ -1,5 +1,4 @@
 import User from "./User.js";
-import Playlist from "./Playlist.js";
-import Song from "./Song.js";
 
-export default [User, Playlist, Song];
+
```

Comment:
Make sure to get your eslint stuff properly configured. Make sure to not have linting errors or else you won't pass the checks